### DISCUSSION TITLE
StarLIte Mk V hardware design issues

### DISCUSSION THREAD

**Discussion by rogercreagh** [StarLabsLtd/firmware#187] [General]
I'm pretty disappointed with some aspects of the detachable keyboard design:

- Using the top of the back as a fold down stand for a unit of this size is a silly idea. When folded into stand position it makes it hard to pick up the combined unit without accidentally detaching it - not good if it is on your lap and you need to put it aside without shutting it
- The magnet which holds the main unit down on the pins is much to weak so that is very easy to dislodge and detached the unit from the keyboard accidentally.
- The hinge itself is just a strip of thin plastic joining the keyboard base to the back and is very floppy. This is a barely satisfactory arrangement on smaller tablets, but here it is a bit of a disaster if trying to use it as a laptop on your lap. The whole thing wobbles around and flexes as you use it.
- The keyboard itself is very flimsy and flexes easily. Coupled with the poor hinge, weak connection, and the klunky stand arrangement, the unit is totally unsuitable for any use except on a table or firm surface.
- Finally the pen is attached to the keyboard in an addon tube of plastic on the left hand side. Apart from being on the wrong side for the 90% of users who are right handed, it is one the wrong unit anyway - it needs to attach to the side of the screen, preferably with a good strong magnet like the pen on my Boox small monochrome tablet that I use as a Kindle and for sketching and handwritten note taking.

Those comments from my longer [blog post](https://blog.crosborne.uk/starlitev-first-impressions) on my first impressions having finally got it last week.

Is there any chance that a new keyboard with a proper hinge might be developed? Can I return my keyboard as not fit for purpose and get my money back to spend on a decent bluetooth keyboard?  (personally I always use a wireless mouse not the touchpad but I like having the touchscreen and pen option as well. I would happily have paid a bit more for a proper detachable keyboard laptop rather than this cheap and not-very cheerful solution

**Comment by Sean-StarLabs** [comment]
Thank you very much for the feedback.

> Using the top of the back as a fold down stand for a unit of this size is a silly idea. When folded into stand position it makes it hard to pick up the combined unit without accidentally detaching it - not good if it is on your lap and you need to put it aside without shutting it

What would you propose instead?

> Is there any chance that a new keyboard with a proper hinge might be developed?

We're looking into it. Ideas being prototyped now are glass surface, the same friction mechanism for the hinge as you call it (so it'll have the solid hinge), stronger magnets and having some form of option pen storage as a lot of people are against the idea of storing it on the tablet.

Cost will skyrocket, but it seems there enough people wanting it regardless of cost, maybe we'll offer two versions.

As for returning, feel free to create a return - you can do it automatically if you received it less than 14-days ago.

**Reply by rogercreagh** [reply]
> What would you propose instead?
> 
> > Is there any chance that a new keyboard with a proper hinge might be developed?
> 
> We're looking into it. Ideas being prototyped now are glass surface, the same friction mechanism for the hinge as you call it (so it'll have the solid hinge), stronger magnets and having some form of option pen storage as a lot of people are against the idea of storing it on the tablet.

That sounds promising except for the glass surface - if you mean not having physical keys with feedback that is not good (my opinion). I'd really want a physical keyboard otherwise you have no means of knowing where your fingers are by feel if you type while looking at the screen not the keyboard. Plus i like the tactile feedback of pressing a key which yields.
 
The friction hinge would do away with the need for a back stand. 

For me I only use the pen when in tablet mode (no keyboard), but not all the time. So the pen storage needs to be on the screen case - the simplest solution is a pen with a flat face attaching magnetically to the side. It can be prone to getting knocked off, but on the Boox I don't find it a big problem. But that would involve a redesign of the screen case so is obviously not possible for retrofit.

I guess the difficult with a friction hinge is having some way of clipping the screen to the backplate so that it didn't pop out when you moved it.

The previous StarLite keyboard was fine, Having moved the guts of the base into the back of the screen it could perhaps be slightly thinner, but without loosing rigidity, with the same hinge and backplate for the screen which would now be thicker and pop out of the backplate. Main problem would be keeping enough mass in the keyboard base to make the combined unit stable.

**Comment by Sean-StarLabs** [comment]
> That sounds promising except for the glass surface - if you mean not having physical keys with feedback that is not good (my opinion). I'd really want a physical keyboard otherwise you have no means of knowing where your fingers are by feel if you type while looking at the screen not the keyboard. Plus i like the tactile feedback of pressing a key which yields.

Glass surface for the trackpad!

> The friction hinge would do away with the need for a back stand.

No, hinges like that can't support almost 1kg that's going to be prodded and poked. I could be wrong, but I think most will favour a kickstand over the bulk of a normal hinge - we shall see.

**Reply by rogercreagh** [reply]
> Glass surface for the trackpad!

Ah, of course. That makes sense (although personally I'd be happier to do away with the trackpad once there is a touchscreen in play - it seems redundant, especially if like me you prefer to use a small wireless mouse. My trackpad is pretty much always turned off - only useful on a non-touchscreen laptop when the mouse battery dies.
> 
> > The friction hinge would do away with the need for a back stand.
> 
> No, hinges like that can't support almost 1kg that's going to be prodded and poked.

Yes, of course you are probably talking with better practical experience than me. I see your point for sure. 

One (equally klunky) alternative to the kickstand would be a kind of folding brace between the keyboard and screen. Would need to be on one side and possibly switchable from left to right according to the users handedness. In its simplest form it would provide a single lock position with the screen at about 100 deg to the keyboard. Most basic kickstands only have one or two fixed positions - your current one does have the benefit of being stiff enough at its fold to provide variable positions.

The big drawback of the current arrangement is that because it folds away from the top it is terribly easy to accidentally remove the screen when picking the whole laplet up to move it in the open position.

At the same time the fold down from the top style is more useable than a fold up from the bottom. Perhaps the fold down stand could leave a backplate supporting the screen and enabling one to pick it up as a single unit. . 

At present I am typing on an ordinary laptop whilst sitting back in an armchair The keyboard is resting on the top of my thighs and lower abdomen. The keyboard is sloping downat about 20 deg with  the screen at about 100 deg to keyboard so that it is about 30deg to vertical but perpendicular to my gaze. My knees are up but the top of the screen is about 3" from the knees. This laptop is stiff enough in the hinge to stay like that - of course using the keyboard and mouse I don't need to prod the screen, but if this was the LiteV then I would have to top fold opened so that it was touching near my knees and that works quite well. If the stand folded up from the bottom like most kickstands it would be useless.

Any way my current idea is to get an old broken laptop with a metal case and rid the guts out and seen if i can mount the LiteV where the screen was and a bluetooth keyboard where the original keyboard was.

As a tablet the LiteV is pretty good, and it does have the potential to make a good laptop as well, but work needs to be done to get there.

**Comment by ghost** [comment]
a lite mkV like the mkIV would be a great device IMO, small laptop with the N200 costing around 500usd. 11,6" are great.

**Comment by Sean-StarLabs** [comment]
It would be closer to $700 - small laptops are very hard to build these days

**Reply by ghost** [reply]
I would prefer it to a 14" laptop. Ideally with a 720p screen to have a better battery life.
